test-support/helpers/interactions.js

```diff
@@ -8,7 +8,7 @@ import {
 
 const { isEmpty } = Ember;
 
-export function checkByLabel(labelText) {
+export function clickByLabel(labelText) {
```

Comment:
I agree that `checkByLabel` probably isn't the best name for a function that calls `click`.

We should probably think about how this function exists alongside [`clickRadioByLabel()`](https://github.com/DockYard/ember-cli-test-interactions/pull/13/files#diff-2243244b3267bbcf7e9edd14d8bbbb93L37)